bin/fixCSpell.ts

```diff
@@ -0,0 +1,17 @@
+import { readFileSync, writeFileSync } from 'node:fs';
```

Comment:
Can you add a quick comment explaining what the file does?

E.g.

```js
/**
 * Sorts all the `words` in the cSpell.json file.
 *
 * Run from the same folder as the `cSpell.json` file
 * (i.e. the root of the Mermaid project).
 */

Reply:
Done :)